projects/addon-doc/src/components/language-switcher/language-switcher.component.ts

```diff
@@ -0,0 +1,32 @@
+import {ChangeDetectionStrategy, Component, Inject} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {TuiLanguageName, TuiLanguageSwitcher} from '@taiga-ui/i18n';
+
+@Component({
+    selector: `tui-language-switcher`,
+    templateUrl: `./language-switcher.component.html`,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TuiLanguageSwitcherComponent {
+    language = new FormControl(this.switcher.currentLanguage);
+
+    flags: Map<TuiLanguageName, string> = new Map([
+        [`Chinese`, `🇨🇳 chinese`],
```

Comment:
These should be capitalized as well 
![image](https://user-images.githubusercontent.com/11832552/182546970-d3677e57-1494-465f-aa7c-ad0be595c1aa.png)

Reply:
> These should be capitalized as well ![image](https://user-images.githubusercontent.com/11832552/182546970-d3677e57-1494-465f-aa7c-ad0be595c1aa.png)

@splincode oh, according to this screenshot, the flag-smiles do not properly work on Windows OS.

May be we should use the same images as inside `InputPhoneInternational` ?
For example, for English:
https://github.com/Tinkoff/taiga-ui/blob/main/projects/icons/src/GB.png

Reply:
fixed